public/donate.html

```
@@ -0,0 +1,667 @@
+      <!-- LEFT — DONATION FORM -->
+      <div class="bg-white dark:bg-gray-800 rounded-2xl shadow-sm border border-gray-200 dark:border-gray-700 p-6 sm:p-8 lg:sticky lg:top-6">
+        <h2 class="text-2xl font-bold text-gray-900 dark:text-gray-100 mb-6">Make a Donation</h2>
+
+        <form id="donate-form" novalidate>
+
+          <!-- Donation Type -->
+          <div class="mb-6">
+            <label class="block text-sm font-semibold text-gray-700 dark:text-gray-300 mb-2">Donation Type</label>
+            <div class="flex rounded-xl overflow-hidden border border-gray-300 dark:border-gray-600">
+              <label class="flex-1 cursor-pointer">
+                <input type="radio" name="donation_type" value="one-time" class="sr-only" checked
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)